api/document/js/__tests__/Api.test.ts

```diff
@@ -51,7 +53,7 @@ describe('fetch()', () => {
     error.response = { data: { oh: 'noes', and: 6 } };
     axios.get = jest.fn(() => { throw error; });
 
-    await api.fetch();
+    try { await api.fetch(); } catch (_) {}
```

Comment:
Since we're raising this exception now, should we explicitly check it? Something like
```
expect(async () => await api.fetch()).toThrow(error);
```

Reply:
Ah good idea, I will bust it up.